packages/syft/src/syft/service/notifier/smtp_client.py

```diff
@@ -4,13 +4,14 @@
 import smtplib
 
 # third party
+from pydantic import BaseModel
 from result import Err
 from result import Ok
 from result import Result
 
 
-class SMTPClient:
-    SOCKET_TIMEOUT = 5  # seconds
+class SMTPClient(BaseModel):
+    SOCKET_TIMEOUT: float = 5  # seconds
```

Comment:
Hey @snwagh! Not `int`?

Reply:
I found a reference that said the `timeout` parameter of smtplib was float but it seems that I was wrong. It is `int32`, will change this, thanks!

Reply:
Resolved [4fc02d9](https://github.com/OpenMined/PySyft/pull/9107/commits/4fc02d99524abf97a58db8fa74aefcdb9cc96feb)